### ISSUE TITLE
MegaSync Crash Repeatedly on Win10 OS Desktop version 

### ISSUE THREAD

**Issue by alirezadvi** [meganz/MEGAsync#448]
I have  problem in Win10 OS, I check my event viewer and crash log is:
Applictaion Error
Faulting application name: MEGAsync.exe, version: 4.3.1.0, time stamp: 0x5e6fe641
Faulting module name: ucrtbase.dll, version: 10.0.17134.677, time stamp: 0x9b002dcc
Exception code: 0xc0000409
Fault offset: 0x000a1aeb
Faulting process id: 0x2290
Faulting application start time: 0x01d62e91ce104af0
Faulting application path: C:\ProgramData\MEGAsync\MEGAsync.exe
Faulting module path: C:\WINDOWS\System32\ucrtbase.dll
Report Id: 8f313e3f-81ae-4a54-ad67-f4589e4d671b
Faulting package full name:
Faulting package-relative application ID:

**Comment by mattw-mega** [comment]
Hi @alirezadvi , 0xc0000409 means STATUS_STACK_BUFFER_OVERRUN. This could occur if you have an extremely deep folder structure inside a sync. Could you please check if that is the case here? By extemely deep I mean more than 200 levels of nested folders. thanks

**Comment by alirezadvi** [comment]
No, I haven't any deep folder in my data, when I want to sync my folder in local with cloud MegaSync crash and I can't reopen it again

**Comment by mattw-mega** [comment]
When you restart MEGAsync after this crash, does it show you a dialog box with some crash details and offer to send the data to MEGA?  If so please do, it will contain the call stack that will give us some clues.   Please put some comments in it before sending, and post those here so we can search for those in the crash reports.  thanks

**Comment by pedrofsantoscom** [comment]
I have this exact same issue happening just a couple hours ago:

```
Faulting application name: MEGAsync.exe, version: 4.3.1.0, time stamp: 0x5e6fe641
Faulting module name: ucrtbase.dll, version: 10.0.18362.815, time stamp: 0xbea5fce0
Exception code: 0xc0000409
Fault offset: 0x0009e6eb
Faulting process id: 0x1170
Faulting application start time: 0x01d6391e02e217a4
Faulting application path: C:\Users\*removed*\AppData\Local\MEGAsync\MEGAsync.exe
Faulting module path: C:\WINDOWS\System32\ucrtbase.dll
Report Id: b44562f5-b060-4eee-9207-dd637cdbf00c
Faulting package full name: 
Faulting package-relative application ID: 
```

I don't see any dialog box when i restart. It keeps crashing. Any ideia what can i do?
I had debug mode active but after looking at the file i don't the log file logs start up issues.

**Comment by pedrofsantoscom** [comment]
The issues solved itself after i got 3 BSOD related to a program i had installed, that gives up&down traffic information. Uninstalled it and it started working without issues.
